artemis-commons/src/main/java/org/apache/activemq/artemis/utils/runnables/AtomicRunnable.java

```diff
@@ -23,8 +23,8 @@
 public abstract class AtomicRunnable implements Runnable {
 
    public static AtomicRunnable checkAtomic(Runnable run) {
-      if (run instanceof AtomicRunnable) {
-         return (AtomicRunnable)run;
+      if (run instanceof AtomicRunnable runnable) {
+         return runnable;
```

Comment:
```suggestion
      if (run instanceof AtomicRunnable atomicRunnable) {
         return atomicRunnable;
```
It is already a Runnable, so calling it merely _runnable_ right after we determined it is more than just that and return it as a different type, simply reads horribly, far more than the original cast.

Reply:
Fixed.